CourseApp/CalculatingAPersonsAge.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace CourseApp
+{
+    using System;
+
+    public class CalculatingAPersonsAge
+    {
+        public string CalculatingAge(DateTime born)
+        {
+            return CalculatingAge(born, DateTime.Today);
+        }
+
+        public string CalculatingAge(DateTime born, DateTime end)
+        {
+            DateTime date = DateTime.MinValue.AddTicks(end.Ticks - born.Ticks);
```

Comment:
also you could use difference between 2 dates and create new by providing difference in constructor